src/pages/docs/administration/reporting/report-on-deployments-using-excel.md

```diff
@@ -7,11 +7,7 @@ description: How to report on deployments using Excel & XML
 navOrder: 10
 ---
 
-:::div{.hint}
-Also take a look at the [Reporting features](/docs/administration/reporting) which provides reporting optimized views of the live deployment data, no longer requiring an XML export.
-:::
```

Comment:
I removed this hint as I found it confusing. It just takes you back to the reporting index and the description doesn't really mean anything. I guess it's suggesting the database approach? You can always just go straight to the database page anyway.